tests/test_check_cloudformation.py

```diff
@@ -0,0 +1,73 @@
+from unittest.mock import patch
+
+import pytest
+from click.testing import CliRunner
+
+import commands
+from commands.check_cloudformation import check_cloudformation as check_cloudformation_command, valid_checks
+
+
+class TestCheckCloudformationCommand:
+    @pytest.fixture
+    def valid_checks_dict(self):
```

Comment:
Is this meant to match the return value of the `valid_checks` method?

Reply:
Yes. We're patching that method so that these tests can check the behaviour of the check command, not the specifics of how it ends up being used.

Idea being that we can add or remove checks, without needing to come back and adjust all these tests.